### PR TITLE
Adding separate `README.md` files for the 00 and 01 examples.

### PR DESCRIPTION
**Pull request by ash-sxn**
- adding separate README.md files for each example as suggested by @berviantoleo and @jmMeessen 
- added for 00 and 01 here and in #14 for 02 example 

 Pinging mentors 
 @gounthar  @berviantoleo @jmMeessen

### CODE REVIEW

**Review by gounthar — APPROVED** [review]

File: experimental_docker_compose_files/00_old_one_from_proposal/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Example 0
+### Old docker-compose file 
+* This is the docker-compose file I created for the proposal, which mimics the steps from the docker section of the Jenkins docker page.
+* This file does not configure any agent.
+* More info about the file can be found in the proposal [here](https://docs.google.com/document/d/1ZpPihadYqpAvR20rxZkTD2SVpf34E6YMzg6opU6yHAg/edit#heading=h.lntg56ljm653).
+### How to run 
+* Make sure you are in the `00_old_one_from_proposal` directory by using the command `pwd`. 
+* Then use the command `docker compose up -d` to run the example.
```
Comment (jmMeessen) [inline experimental_docker_compose_files/00_old_one_from_proposal/README.md:8]: This doesn't work on my machine (at least as long as the compose file is prefixed with 00). Machine is Mac OS with  Docker 20.10.10.
```
 01_simple_controller+agent> docker compose up
can't find a suitable configuration file in this directory or any parent: not found
```

To make it work, I need to specify the `-f` with the exact name of the compose file.. We could either rename the compose file or update the documentation.

I vote for renaming the compose file.

File: experimental_docker_compose_files/01_simple_controller+agent/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Example 1
+### First docker-compose file of the project 
+* This is a simple docker-compose file that runs two containers, a Jenkins controller and an agent, and gives the agent the public ssh key as an environment variable.
+* However, the agent needs to be connected manually to the container from the UI.
+* The SSH key used can be found in the ../ssh_keys directory.
+### How to run 
+* Make sure you are in the `01_simple_controller+agent` directory by using the command `pwd`. 
+* Then use the command `docker compose up -d` to run the example.
```
Comment (jmMeessen) [inline experimental_docker_compose_files/01_simple_controller+agent/README.md:8]: Same remark as for the previous directory. I recommend renaming the compose file for a canonical format.

**Review by jmMeessen — COMMENTED** [review]
I recommend adding a cleanup instruction. Extra steps are required to remove the jenkins_home named volume and this volume can come in the way when running the examples one after the other.

**Review by gounthar — CHANGES_REQUESTED** [review]
Please change your file naming according to @jmMeessen 's comments.

File: experimental_docker_compose_files/00_old_one_from_proposal/README.md (hunk 1 above)
Comment (ash-sxn) [reply experimental_docker_compose_files/00_old_one_from_proposal/README.md:8]: I've renamed the files

Comment (berviantoleo) [reply experimental_docker_compose_files/00_old_one_from_proposal/README.md:8]: I think we can mention this issue: https://github.com/ash-sxn/GSoC-2023-docker-based-quickstart/issues/21

**Review by berviantoleo — APPROVED** [review]

**Review by gounthar — CHANGES_REQUESTED** [review]
Please add cleanup instructions as requested by @jmMeessen .

File: experimental_docker_compose_files/01_simple_controller+agent/README.md [hunk 3]
```diff
@@ -6,3 +6,6 @@
 ### How to run 
 * Make sure you are in the `01_simple_controller+agent` directory by using the command `pwd`. 
 * Then use the command `docker compose up -d` to run the example.
```
Comment (gounthar) [inline experimental_docker_compose_files/01_simple_controller+agent/README.md:8]:
```suggestion
* Then use the command `docker compose up -d --build` to run the example.
```

File: experimental_docker_compose_files/01_simple_controller+agent/README.md [hunk 4]
```diff
@@ -6,3 +6,6 @@
...
+### Clean Up Instructions 
+* To clean, run `docker compose down`. This will remove the containers and network.
```
Comment (gounthar) [inline experimental_docker_compose_files/01_simple_controller+agent/README.md:10]:
```suggestion

* To stop and clean up a working Docker Compose setup, you can use the following command:
`docker compose down` 

This command stops the containers defined in your Docker Compose file and removes them along with any networks, volumes, and other resources created by the Compose setup.
It effectively cleans up the resources associated with your Docker Compose environment.

By default, `docker compose down` removes the stopped containers and networks, but it preserves named volumes.
If you also want to remove the named volumes, you can use the `--volumes` flag:
`docker compose down --volumes` 

This ensures that all resources, including volumes, are removed.

Note that the `docker compose down` command should be executed from the same directory where your `docker-compose.yml` file is located.
```

**Review by gounthar — APPROVED** [review]

**Review by jmMeessen — APPROVED** [review]
LGTM

**Pull request merged**